addons/controllers/testdata/test-cluster-bootstrap-2.yaml

```diff
@@ -28,47 +20,195 @@ spec:
       cidrBlocks: [ "192.168.0.0/16","fd00:100:96::/48" ]
     services:
       cidrBlocks: [ "192.168.0.0/16","fd00:100:96::/48" ]
-  topology:
-    class: test-clusterclass-tcbt
-    version: v1.21.2---vmware.1-tkg.1
 ---
-apiVersion: run.tanzu.vmware.com/v1alpha3
-kind: ClusterBootstrapTemplate
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
 metadata:
-  name: v1.21.2---vmware.1-tkg.1
-  namespace: tkg-system
+  name: foobar.example.com.1.17.2
+  namespace: cluster-namespace-2
 spec:
-  cnis:
-    - refName: "calico.tanzu.vmware.com.0.3.0--vmware.1-tkg.1-rc.1"
-      valuesFrom:
-        providerRef:
-          apiGroup: cni.tanzu.vmware.com
-          kind: CalicoConfig
-          name: test-cluster-tcbt-2
-    - refName: "antrea.tanzu.vmware.com.21.2.0--vmware.1-tkg.1-rc.1"
-      valuesFrom:
-        providerRef:
-          apiGroup: cni.tanzu.vmware.com
-          kind: AntreaConfig
-          name: test-cluster-tcbt-2
-
```

Comment:
Why removing all these? Isn't that why CNIs tests have been failing and now commented?

Reply:
that is because when envtest get set up for clsuterbootstrap_controller_test.go, test-cluster-bootstrap-1.yaml has already contains those resources and get deployed. No need to duplicate them here.